textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
+TextSubs.add('"Stone Strike."', '"Stone Strike(Warrior Mage)."')
+TextSubs.add('"Tremor."', '"Tremor(Warrior Mage)."')
+TextSubs.add('"Anther\'s Call."', '"Anther\'s Call(Warrior Mage)."')
+TextSubs.add('"Sure Footing."', '"Sure Footing(Warrior Mage)."')
+TextSubs.add('"Magnetic Ballista."', '"Magnetic Ballista(Warrior Mage)."')
+TextSubs.add('"Ring of Spears."', '"Ring of Spears(Warrior Mage)."')
+TextSubs.add('"Aegis of Granite."', '"Aegis of Granite(Warrior Mage)."')
+TextSubs.add('"Lightning Bolt."', '"Lightning Bolt(Warrior Mage)."')
+TextSubs.add('"Gar Zeng."', '"Gar Zeng(Warrior Mage)."')
+TextSubs.add('"Arc Light."', '"Arc Light(Warrior Mage)."')
+TextSubs.add('"Tingle."', '"Tingle(Warrior Mage)."')
+TextSubs.add('"Chain Lightning."', '"Chain Lightning(Warrior Mage)."')
+TextSubs.add('"Electrostatic Eddy."', '"Electrostatic Eddy(Warrior Mage)."')
+TextSubs.add('"Grounding Field."', '"Grounding Field(Warrior Mage)."')
+TextSubs.add('"Fire Shards."', '"Fire Shards(Warrior Mage)."')
+TextSubs.add('"Fire Ball."', '"Fire Ball(Warrior Mage)."')
+TextSubs.add('"Ignite."', '"Ignite(Warrior Mage)."')
+TextSubs.add('"Dragon\'s Breath."', '"Dragon\'s Breath(Warrior Mage)."')
+TextSubs.add('"Fire Rain."', '"Fire Rain(Warrior Mage)."')
+TextSubs.add('"Mark of Arhat."', '"Mark of Arhat(Warrior Mage)."')
+TextSubs.add('"Mantle of Flame."', '"Mantle of Flame(Warrior Mage)."')
+TextSubs.add('"Geyser."', '"Geyser(Warrior Mage)."')
+TextSubs.add('"Frost Scythe."', '"Frost Scythe(Warrior Mage)."')
+TextSubs.add('"Frostbite."', '"Frostbite(Warrior Mage)."')
+TextSubs.add('"Ice Patch."', '"Ice Patch(Warrior Mage)."')
+TextSubs.add('"Rising Mists."', '"Rising Mists(Warrior Mage)."')
+TextSubs.add('"Rimefang."', '"Rimefang(Warrior Mage)."')
+TextSubs.add('"Veil of Ice."', '"Veil of Ice(Warrior Mage)."')
+TextSubs.add('"Fortress of Ice."', '"Fortress of Ice(Warrior Mage)."')
+
+TextSubs.add('"Breath of Storms."', '"Breath of Storms(Bard)."')
+TextSubs.add('"Echoes of Aether."', '"Echoes of Aether(Bard)."')
+TextSubs.add('"Will of Winter."', '"Will of Winter(Bard)."')
+TextSubs.add('"Words of the Wind."', '"Words of the Wind(Bard)."')
+TextSubs.add('"Soul Ablaze."', '"Soul Ablaze(Bard)."')
+TextSubs.add('"Phoenix\'s Pyre."', '"Phoenix\'s Pyre(Bard)."')
+TextSubs.add('"Desert\'s Maelstrom."', '"Desert\'s Maelstrom(Bard)."')
+TextSubs.add('"Caress of the Sun."', '"Caress of the Sun(Bard)."')
+TextSubs.add('"Misdirection."', '"Misdirection(Bard)."')
+TextSubs.add('"Redeemer\'s Pride."', '"Redeemer\'s Pride(Bard)."')
+TextSubs.add('"Rage of the Clans."', '"Rage of the Clans(Bard)."')
+TextSubs.add('"Abandoned Heart."', '"Abandoned Heart(Bard)."')
+TextSubs.add('"Damaris\'s Lullaby."', '"Damaris\'s Lullaby(Bard)."')
+TextSubs.add('"Albreda\'s Balm."', '"Albreda\'s Balm(Bard)."')
+TextSubs.add('"Faenella\'s Grace."', '"Faenella\'s Grace(Bard)."')
+TextSubs.add('"Nexus."', '"Nexus(Bard)."')
+TextSubs.add('"Beckon the Naga."', '"Bacon the Naga(Bard)."')
```

Comment:
On the one hand, I like easter eggs, on the other hand... not everyone would realize _why_ this sub happened, so there could be a bad user experience there...